core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -676,4 +677,24 @@ default EventExecutor defaultSubscriberExecutor() {
         requireNonNull(function, "function");
         return new RecoverableStreamMessage<>(this, function, /* allowResuming */ true);
     }
+
+    /**
+     *  Provide general way to use {@link StreamMessages} by applying the specified {@link Function}.
+     *
+     *  <p>Example:<pre>{@code
+     *  ByteBuf[] bufs = new ByteBuf[10];
+     *  for(int i=0; i<10; i++){
+     *             bufs[i] = Unpooled.wrappedBuffer(Integer.toString(i).getBytes());
+     *   }
+     *  StreamMessage<ByteBuf> streamMessage = StreamMessage.of(bufs);
+     *  streamMessage.writeTo( x -> HttpData.wrap())
+     *  }</pre></p>
+     */
+    default CompletableFuture<Void> writeTo(
+            Function<? super T, ? extends HttpData> mapper, Path destination, OpenOption... options) {
```

Comment:
```suggestion
    default CompletableFuture<Void> writeTo(Function<? super T, ? extends HttpData> mapper, 
                                            Path destination, OpenOption... options) {
```
nit. use same indent with `StreamMessages#writeTo`

Reply:
> nit. use same indent with `StreamMessages#writeTo`

ㄹㄹ